### PR TITLE
Add new messages for iono and l2c capabilities

### PR DESCRIPTION
**Pull request by pmiettinen**

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/observation.yaml [hunk 1]
```diff
@@ -946,3 +946,46 @@ definitions:
...
+          type: double
+          units: s/semi-circle
+      - b2:
+          type: double
+          units: s/(semi-circle)^2
+      - b3:
+          type: double
+          units: s/(semi-circle)^3
+
+ - MSG_SV_CONFIGURATION_GPS:
```
Comment (ljbade) [inline spec/yaml/swiftnav/sbp/observation.yaml]: I recall we are also capturing the L2C group delays?

Comment (pmiettinen) [inline spec/yaml/swiftnav/sbp/observation.yaml]: Oh yes, need to add those too.

**Comment by ljbade** [comment]
Per discussion in planning ticket, can you add a t_nmct (gps_time_t) to iono and L2C capabilities message. And t_op (gps_time_t) to L2C group delay message.

**Comment by ljbade** [comment]
I think these messages are good now.

**Comment by JoshuaGross** [comment]
This looks good to me. Merge at will.

**Comment by JoshuaGross** [comment]
Actually, I know y'all want it merged, I'll just do it for you :)

**Pull request merged**